sdk/storage/azblob/sas/service.go

```diff
@@ -269,8 +269,8 @@ func getCanonicalName(account string, containerName string, blobName string, dir
 // Initialize an instance of this type and then call its String method to set BlobSignatureValues' Permissions field.
 // All permissions descriptions can be found here: https://docs.microsoft.com/en-us/rest/api/storageservices/create-service-sas#permissions-for-a-directory-container-or-blob
 type ContainerPermissions struct {
-	Read, Add, Create, Write, Delete, DeletePreviousVersion, List, FilterByTags, Move, SetImmutabilityPolicy bool
-	Execute, ModifyOwnership, ModifyPermissions                                                              bool // Meant for hierarchical namespace accounts
+	Read, Add, Create, Write, Delete, DeletePreviousVersion, List, Tags, FilterByTags, Move, SetImmutabilityPolicy bool
```

Comment:
As per the blow docs, the tags(t) permission is for blob resource only, not container.
https://learn.microsoft.com/en-us/rest/api/storageservices/create-service-sas#permissions-for-a-directory-container-or-blob
https://learn.microsoft.com/en-us/azure/storage/blobs/storage-manage-find-blobs?tabs=azure-portal#service-sas-for-a-blob

Is it correct to add tags in the ContainerPermissions? How is this in other SDKs?

Reply:
This is settable in the Java SDK https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/sas/BlobContainerSasPermission.java#L32

Also the live test works, so the service does support it.